docs: updated animation docs intro section and example (#42046)
angular/angular: aio/content/guide/animations.md

  <img src="generated/images/guide/animations/open-closed.png" alt="open and closed states">
</div>

<div class="alert is-helpful">

  Let's create a new `open-close` component to animate with simple transitions.

  Run the following command in terminal to generate the component

  `ng g component open-close`

> </div>

Reviewer: Shall we add something like the following to make it clear about the new files that are going to be referenced?

```
This will create the component at `src/app/open-close.component.ts`.
```

Author: Yes, definitely yes. Some time if any new developer is using this guide it will be more clear for them. Let me update it.

It's great suggestion.

  <img src="generated/images/guide/animations/open-closed.png" alt="open and closed states">
</div>

<div class="alert is-helpful">

>   Let's create a new `open-close` component to animate with simple animations.

Reviewer: ```suggestion
  Let's create a new `open-close` component to animate with simple transitions.
```

>   Run below command in terminal to generate component

Reviewer: ```suggestion
  Run the following command in terminal to generate the component
```

>   Run the following command in terminal to generate the component

Reviewer 2: ```suggestion
  Run the following command in terminal to generate the component:
```

  `ng g component open-close`

>   We will be using this component in introduction section of animation. One can use above command to generate component in other sections as well if they find any new component name being used.

Reviewer 3: np, I will remove this  paragraph.
